_ont/ont-huawei-ma5671a-root.md

```diff
@@ -133,9 +137,8 @@ The password is `admin123`.
 {% include alert.html content="If you use a modern OpenSSH version (e.g. >= 8.8) you will have to enable some deprecated algorithms: ssh -oKexAlgorithms=+diffie-hellman-group1-sha1 -oHostKeyAlgorithms=+ssh-dss [...]" alert="Info"  icon="svg-info" color="blue" %}
 
 # Miscellaneous Links
+- [Latest version of Root Procedure (V1) on hack-gpon](https://github.com/hack-gpon/hack-gpon.github.io/blob/b0edc817a49d42c9ab3a5c096944caeb0a812155/_ont/ont-huawei-ma5671a-root.md)
```

Comment:
```suggestion
- [Latest version of Root Procedure (V3) on hack-gpon](https://github.com/hack-gpon/hack-gpon.github.io/blob/b0edc817a49d42c9ab3a5c096944caeb0a812155/_ont/ont-huawei-ma5671a-root.md)
```

Reply:
no this link redirects to an old version of the root procedure, V1 is a Retronym for the first procedure, the one with the tweezers